src/Microsoft.DotNet.ProjectModel/ProjectContextBuilder.cs

```diff
@@ -37,6 +37,8 @@ public class ProjectContextBuilder
 
         private Func<string, LockFile> LockFileResolver { get; set; }
 
+        private ProjectReader.Settings Settings { get; set; }
```

Comment:
Let's create another issue for refactoring existing code. We need to get this done pretty soon, tho.